### DISCUSSION TITLE
Way to render the i18n-key for all translated texts

### DISCUSSION THREAD

**Discussion by trm217** [vinissimus/next-translate#915] [Q&A] [answered]
Perhaps I missed this in the docs, but is there a built in way to show the i18n-key for all texts on the site instead of the actual translations?
This would really help while in the process of translating the page, by seeing all non i18n-texts.

**Comment by aralroca** [comment, marked as answer]
The default fallback when no translation exists is to display the key. So if you add a new language that has no translation you will be able to see all the keys on the page.
